docs/docs/getting-started.md

```diff
@@ -17,29 +17,41 @@ To address the Gateway you can either compile the Protobuf files yourself or use
 
 The easiest way to use the Jalapeño API Gateway is to simply download and import the pre-compiled, language-specific Protobuf files from the release page: 
 
-- [Protorepo](https://github.com/jalapeno-api-gateway/protorepo/releases/latest)
+- [JAGW](https://github.com/jalapeno-api-gateway/jagw/releases/latest)
 
 #### Option 1.2: Golang only
 
 If you are using Golang, you will not find the pre-compiled files in the release page, but rather in a dedicated repository:
 
-- [Golang](https://github.com/jalapeno-api-gateway/protorepo-jagw-go/releases/latest)
+- [JAGW-Go](https://github.com/jalapeno-api-gateway/jagw-go/releases/latest)
 
 This allows you to directly import the go module to your project with:
 
 ```bash
-$ go get github.com/jalapeno-api-gateway/protorepo-jagw-go@v1.3.0
+$ go get github.com/jalapeno-api-gateway/jagw-go@v1.3.2
+```
+
+#### Option 1.3: Python only
+
+If you are using Python, you will not find the pre-compiled files in the release page, but rather in a dedicated repository:
+
+- [JAGW-Python](https://github.com/jalapeno-api-gateway/jagw-python/releases/latest)
```

Comment:
I suggest we're giving the link to the repo and not the empty releases:
```suggestion
- [JAGW-Python](https://github.com/jalapeno-api-gateway/jagw-python/)
```

Reply:
We will create a release. Therefore, I suggest leaving the link as it is.